openssl-dynamic/src/main/c/ssl.c

```diff
@@ -2113,3 +2043,7 @@ TCN_IMPLEMENT_CALL(jbyteArray, SSL, getOcspResponse)(TCN_STDARGS, jlong ssl) {
     return value;
 #endif
 }
+
+void tcn_ssl_unload() {
+    ssl_init_cleanup();
```

Comment:
can you combine `tcn_ssl_unload` and `ssl_init_cleanup`? seems unnecessary to have a function which only calls a single function?